Shokofin/Sync/UserDataSyncManager.cs

```diff
@@ -444,7 +454,14 @@ private async Task SyncVideo(Video video, UserConfiguration userConfig, SyncDire
                 return;
             }
             var localUserStats = UserDataManager.GetUserData(userConfig.UserId, video);
-            var remoteUserStats = await APIClient.GetFileUserStats(fileId, userConfig.Token);
+            UserStats remoteUserStats;
```

Comment:
If we're going to add a `try...catch` anyway then please wrap the whole method body in it instead of the first usage of the api client. Even if it's _a lot rarer and most likely purely an edge case_, **but**, we do potentially send a PUT request (in multiple switch branches too) after the initial GET request that may also fail. Though that's highly unlikely it's still a possibility.